apps/studio/tests/integration/helpers/seed/index.ts

```diff
@@ -486,7 +486,7 @@ export const setupUser = async ({
   userId?: string
   email: string
   phone?: string
-  isDeleted: boolean
+  isDeleted?: boolean
```

Comment:
thanks for changing this haha been wanting to change this since it was quite annoying but kept forgetting